app/components/sy-modal/template.hbs

```diff
@@ -1,12 +1,16 @@
-<EmberWormhole @to="sy-modals">
-  <SyModal::Overlay @visible={{@visible}} @on-close={{this.close}}>
-    <div class="modal-dialog {{if @size (concat 'modal-dialog--' @size)}}">
-      {{yield (hash
-        header = (component 'sy-modal/header' close=this.close)
-        body   = (component 'sy-modal/body'   close=this.close)
-        footer = (component 'sy-modal/footer' close=this.close)
-        close  = (this.close)
-      )}}
-    </div>
-  </SyModal::Overlay>
-</EmberWormhole>
+{{#if @visible}}
```

Comment:
Why is this needed? SyOverlay already seems to have an @visible

Reply:
The modal was still hanging in the DOM even though it was invisible. This way it will get cleaned properly. The component will get refactored for ember v4 anyways. :shrug: